Library/Homebrew/cmd/update.rb

```diff
@@ -263,27 +263,43 @@ def pull!(options = {})
     @current_revision = read_current_revision
 
     if @initial_branch != "master" && !@initial_branch.empty?
```

Comment:
Now that you're touching this piece of code, shouldn't that be `@initial_branch != @upstream_branch`?